ansible_runner/utils/streaming.py

```diff
@@ -79,6 +84,11 @@ def unstream_dir(stream, length, target_directory):
 
                 is_symlink = mode[:1] == 'l'
                 if os.path.exists(out_path):
+                    if stat.S_ISFIFO(os.stat(out_path).st_mode):
+                        # must remove any pipes before reading
+                        # i.e. ssh_key_data that was never cleaned up
+                        os.remove(out_path)
```

Comment:
This one makes more sense to delete as a safety, esp since it will only trigger when the archive contains a file with the same name/path- seems a lot less likely to stomp an arbitrary socket file laying around that might be in use...